src/contracts/atlas/Storage.sol

```diff
@@ -52,12 +52,14 @@ contract Storage is AtlasEvents, AtlasErrors, AtlasConstants {
     uint256 internal S_cumulativeSurcharge; // Cumulative gas surcharges collected
     address internal S_surchargeRecipient; // Fastlane surcharge recipient
     address internal S_pendingSurchargeRecipient; // For 2-step transfer process
+    address internal S_l2GasCalculator; // Gas calculator for L2s
```

Comment:
I think we should make it an immutable var as there's no way to set it after deployment anyway. Will do that quick

Reply:
good idea  (otherwise is would be useful to add functionality to update the address)